src/Tableau/branchAndCut.js

```diff
@@ -80,9 +82,16 @@ Tableau.prototype.branchAndCut = function () {
     // 1.) Load a model into the queue
     var branch = new Branch(-Infinity, []);
     branches.push(branch);
-
     // If all branches have been exhausted terminate the loop
-    while (branches.length > 0) {
+    while (branches.length > 0 && toleranceFlag === true) {
+        var acceptableThreshold = this.bestPossibleEval * (1 - (tolerance/100));
```

Comment:
I might be wrong but I think that in order to get the best out of this feature, you need to update ```bestPossibleEval``` after the continues blocks (so at line 120) with ```evaluation```.